app/controllers/api/v1/users_controller.rb

```diff
@@ -20,6 +20,7 @@ class UsersController < Api::V1::ApiController
         For a donor, this will be just themselves. For administrators, this will be all users.
       EOS
       def index
+        @users = @users.stock_users if bool_param(:stockUsers, false)
```

Comment:
We generally want to move away from `per app` server design. Everything should be app-agnostic.
What you can do is request per role, or per permission. Which wouldn't involve any app name

Reply:
here `stock_users` scope will fetch user having roles `["Supervisor", "Reviewer", "Order fulfilment", "Order administrator"]`
and if `stockUsers` param value is true, it will give users with above roles only..

Should I just rename the param attribute `stockUsers` to something else? or some other way you are suggesting @patrixr

Reply:
I would suggest an API filter

`GET /users?roles=supervisor,reviewer,order_fulfilment,...`

That would allow any app to filter the user for any purpose.